core/tests/protractor/explorationFeedback.js

```diff
@@ -62,9 +64,9 @@ describe('ExplorationFeedback', function() {
     // Creator creates and publishes an exploration
     users.login('user1@ExplorationFeedback.com');
     workflow.createAndPublishExploration(EXPLORATION_TITLE,
```

Comment:
Consider moving the first arg to the next line.

Reply:
Got it!